draft-ietf-trans-rfc6962-bis.md

```diff
@@ -552,14 +552,12 @@ Tree no later than a fixed amount of time, known as the Maximum Merge Delay
 (MMD), after the issuance of the SCT. Periodically, the log MUST append all its
 new entries to its Merkle Tree and sign the root of the tree.
 
-Log operators MUST NOT impose any conditions on retrieving or sharing data from
```

Comment:
Do I recall incorrectly that the suggestion was to change the MUST to a SHOULD here? 
Overall I agree this is a policy matter, but making a suggestion here (in a normative or an informative manner) seems like a good idea.